.github/workflows/repo-dispatch.yaml

```diff
@@ -17,6 +17,10 @@ on:
         description: "target repository"
         required: true
         type: string
+      repository-owner:
+        description: "target repository owner"
+        required: false
```

Comment:
Shouldn't this be required? Otherwise how can the input be passed down as `inputs.repository-owner`?
What about setting a default value for the "default owner" e.g. `strg-at`?